test/endpoint_testrunner.py

```diff
@@ -0,0 +1,238 @@
+import os
```

Comment:
any reason this file can't go inside sm endpoints test directory?

Reply:
I recommend running a linter (like pylint) on this file to check for syntax/style/formatting errors

Reply:
I think it is at the right place just like the testrunner.py. This is another type of test runner we now have.

Reply:
In my opinion, these are effectively not tests for DLC release candidates. I would encapsulate this all under one directory for ease of removal